### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Shikkic**
Left some notes, everything looks great! Lets talk about the roadmap for the app asap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,27 +12,43 @@ Complete iOS feature set:
...
+    + upvote button to upboat your current location (Picture REQUIRED) 
     + press and hold to bring up the camera, release to take a picture
         * alternatively tap to bring up camera, tap to take picture
+        * Make sure cannot select pic from library
     + upload picture to s3, send s3 url to database along with upvote
     + Realm-powered live map updating with nearby bitters
-        * filter by `All`, `Friends`, and `Mine`
+        * filter by `All`, `Friends` or `Friend`, and `Mine`
+    + All map functionality should live in one view 
+    + Upboat other uboats
```
Comment (shrugs) [inline README.md:23]: Just to make sure, users can only upboat a Bitter when they're near the bitter's location?

Comment (shrugs) [inline README.md:23]: @Shikkic

File: README.md [hunk 2]
```diff
@@ -12,27 +12,43 @@ Complete iOS feature set:
...
+    + Implemant Decay Algo 
+        * Example: Biter will decay win x days unless there is y action
+    + Possible build:
+        * Basic photo editor / text inserter (*can skip for first interation)
```
Comment (shrugs) [inline README.md:27]: Definitely a v2 feature, but won't be hard to add; [FastttCamera](https://github.com/IFTTT/FastttCamera) supports filters and I can use [Jot](https://github.com/IFTTT/jot) to add snapchat-esq features.

File: README.md [hunk 3]
```diff
@@ -12,27 +12,43 @@ Complete iOS feature set:
...
 - Profile View
-    + either the picture from my last bitter or a map of my recent bitters at the top
+    + either the picture from my last bitter or a map of my recent bitters at the top (Solid feature) 
     + link to my personal bitters (list view, detail view is a map w/info/photo)
+    + Follow a user
+    + Private / Public user
```
Comment (shrugs) [inline README.md:32]: Is this necessary? I know it's a big deal if we get popular, but for now we should stick with an "everything is public" mindset, no?

File: README.md [hunk 4]
```diff
@@ -12,27 +12,43 @@ Complete iOS feature set:
...
 Random Notes for the Future:
 
 - our refresh slider should _definitely_ be a stretching bitter logo mmmmm
+    * Could be lemon going down on juicer
 - actually implement the MGLMapboxMetricsEnabled option in the app
 - put all of the map attribution information in the settings of the app
-- before open sourcing the app, remove my MapBox key from info.plist
\ No newline at end of file
+- before open sourcing the app, remove my MapBox key from info.plist
+- Do we want to open source / should make sure we don't open source any auth tokens for back end
```
Comment (shrugs) [inline README.md:54]: @Shikkic use environment variables and distribute a `.env` file (that's excluded from git via `.gitignore`) and we'll be fine

File: README.md (hunk 1 above)
Comment (Shikkic) [inline README.md:23]: Oh shit thats a good question, let me clarify with team first

File: README.md (hunk 2 above)
Comment (Shikkic) [inline README.md:27]: Awesome, glad to hear that won't be a blocker

File: README.md (hunk 3 above)
Comment (Shikkic) [inline README.md:32]: Yes, for mvp let's go all public, we could migrate our db with a new private property later if we wanted

File: README.md (hunk 4 above)
Comment (Shikkic) [inline README.md:54]: :+1:

File: README.md (hunk 3 above)
Comment (sup) [inline README.md:32]: Yeah - its more of a later thing

File: README.md (hunk 2 above)
Comment (sup) [inline README.md:27]: oh sweet

**Pull request closed without merging**